client/core/core.go

```diff
@@ -2692,6 +2692,42 @@ func (c *Core) DiscoverAccount(dexAddr string, appPW []byte, certI interface{})
 	return dc.exchangeInfo(), true, nil
 }
 
+// EstimateRegistrationTxFee provides an estimate for the tx fee needed to
+// pay the registration fee for a certain asset. The dex host is required
+// because the dex server is used as a fallback to determine the current
+// fee rate in case the client wallet is unable to do it.
+func (c *Core) EstimateRegistrationTxFee(host string, certI interface{}, assetID uint32) (uint64, error) {
+	wallet, err := c.connectedWallet(assetID)
+	if err != nil {
+		return 0, err
+	}
+
+	var rate uint64
+	if rater, is := wallet.Wallet.(asset.FeeRater); is {
+		if r, err := rater.FeeRate(); err == nil {
+			rate = r
+		} else {
+			c.log.Warnf("failed to get fee suggestion from %s FeeRater: %v", unbip(assetID), err)
+		}
+	}
```

Comment:
I'm on the fence on whether this should actually be a `feeSuggestionAny` to avoid a temp connection if there are existing connections from which we can get a fee rate.

Reply:
I was thinking it made sense just to get the rate from the server that the user is registering to, but both are fine. I'll update if you prefer `feeSuggestionAny`.